### ISSUE TITLE
[Feature request] Option to hide the tile windows icon in the top bar

### ISSUE THREAD

**Issue by 31337-4554551n** [pop-os/gnome-shell-extension-pop-cosmic#208]
The icon in the titlebar that looks like 3 windows needs to be hideable, it takes up a lot of real estate for something that can be handled with ctrl+y

I would also really appreciate it if someone has some back end config I could use to hide the icon until a gui option becomes available.

**Comment by 31337-4554551n** [comment]
Also, wondering if there is a config I can change in the back end to hide that in a less user friendly manner, as long as it's gone, but the functionality remains with the hotkey

**Comment by britshickhouse** [comment]
Yes please for this feature request. Takes up space for a feature I never use.

Or... does anyone have any tips on how to hide/remove that icon?

In the meantime, I'm using Extensions Manager > Browse > Icon Hider Updated (by lx358hcl)
